lib/lib.go

```diff
@@ -193,6 +194,14 @@ func OptCheckConfigMigrations(cfgPath string) Option {
 	}
 }
 
+// OptSetLogAll sets the logAll value so that logging is enabled for all packages
```

Comment:
I'd love a note that mentions this turns on _debug level logging_ for all _qri_ packages. go-log is also connected to IPFS.

Reply:
Done.